lib/processors/text-html/twitter-profile.js

```diff
@@ -0,0 +1,95 @@
+
+/**
+ * Dependencies
+ */
+
+var htmlParser = require('magnet-html-parser/lib/parse');
+var toDom = require('magnet-html-parser/lib/html-parser');
+var debug = require('debug')('twitter-profile');
+var PATH = require('path');
+var url = require('url');
+
+module.exports = function(res) {
+  var doc = toDom(res.body);
+
+  return htmlParser(doc, res, ['icon', 'manifest'])
+    .then(result => {
+      debug('parsed', result);
+      var twitter = parse(doc, res);
+      result.type = 'profile';
+      result.twitter = twitter;
+      result.title = twitter.avatar.alt;
+      result.description = twitter.bio;
+      result.image = twitter.avatar.src;
+      result.icon = twitter.avatar_small.src;
+      return result;
+    });
+};
+
+function parse(doc, res) {
+  var result = {};
+
+  try {
+    var links = doc.querySelectorAll('link[rel="alternate"]');
+    if (links) {
```

Comment:
There is potential for the server to be DDoS'd if we iterate exhaustively, imagine a crafted URL with 1000000s of `link`s - we might want to do something better?